lib/elixir_search_extractor/accounts.ex

```diff
@@ -0,0 +1,350 @@
+defmodule ElixirSearchExtractor.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+
+  import Ecto.Query, warn: false
+  alias ElixirSearchExtractor.Repo
+  # credo:disable-for-next-line Credo.Check.Consistency.MultiAliasImportRequireUse
```

Comment:
Why do we need to disable credo here?

Reply:
@hanam1ni for this issue we have to disable it for now - https://github.com/rrrene/credo/issues/441